docs/testnet/obscuroscan.md

```diff
@@ -4,15 +4,15 @@
 ObscuroScan allows you to decrypt rollup transactions blobs on Testnet. You can also monitor in realtime the L1 blocks and the Obscuro rollups via an Obscuro node connected to the Testnet.
 
 ## How to Monitor L1 Blocks and Obscuro Rollups
-From the [ObscuroScan landing page](http://obscuroscan-01.uksouth.azurecontainer.io/) click _Monitor L1 blocks and Obscuro rollups via the connected Obscuro node_ to go to the monitoring page.
+1. From the [ObscuroScan landing page](http://obscuroscan-01.uksouth.azurecontainer.io/) click _Monitor L1 blocks and Obscuro rollups via the connected Obscuro node_ to go to the monitoring page.
 
-You will see in realtime the current head L1 block and the current Obscuro rollup.
+1. Observe in realtime the current head L1 block and the current Obscuro rollup.
 
-Notice the _Encrypted Transaction Blob_ field. You can copy and paste the field contents into ObscuroScan's transaction blob decryption page to see the content of the transaction in unencrypted plain text. This is a feature of Testnet only.
+Notice the _Encrypted Transaction Blob_ field. You can copy and paste the field contents into ObscuroScan's _Decrypt rollup transaction blobs (testnet only!)_ page to see the content of the transaction in unencrypted plain text. This is a feature of Testnet. In contrast, on Mainnet, rollups will use rotating keys that are not known to anyone, or anything, other than the Obscuro enclaves.
 
 ## How to Decrypt Transaction Blobs
-Decrypting transaction blobs is only possible on Testnet to help you understand how Obscuro works. ObscuroScan on Testnet uses a rollup encryption key which is long-lived and well-known. On Mainnet, rollups will use rotating keys that are not known to anyone, or anything, other than the Obscuro enclaves.
+Decrypting transaction blobs is only possible on Testnet to help you understand how Obscuro works. ObscuroScan on Testnet uses a rollup encryption key which is long-lived and well-known. In contrast, on Mainnet, rollups will use rotating keys that are not known to anyone, or anything, other than the Obscuro enclaves.
```

Comment:
The sentence starting "In contrast, ..." is repeated from the previous paragraph.

Reply:
ok